src/css/tabs/osd.css

```diff
@@ -517,18 +528,28 @@ button {
   }
 }
 
-.tab-osd .grid-col.col4:first-child, .tab-osd  .grid-col.col4:last-child {
-    width: calc((100% - 390px) / 2);
-}
-
 .tab-osd .grid-row {
     margin-bottom: 40px;
 }
 
-.tab-osd .grid-col.col4 {
-    width: auto;
+.tab-osd .grid-row .elements {
+  flex: 1 1 auto;
+  width: min-content;
+  max-width: 700px;
 }
 
+.tab-osd .grid-row .osd-preview {
+  flex: 2 1 min-content;
```

Comment:
Invalid property according to chrome devtools:

![Screenshot from 2022-05-09 22-17-44](https://user-images.githubusercontent.com/8344830/167490597-17018616-2b31-48e5-a0dc-74d0d820e9d6.png)

Changing `min-content` to `auto` fixes display on Ubuntu

Reply:
Change still works on Windows (but chrome devtools does not complain here)

Reply:
Modified directly to `flex: 2 0 360px` and removed the min-width. I think works better.

Reply:
Sorry but on Linux this does not work. Leaving out the last parameters works on Linux and also on WIndows on the 3 grid columns.
```
flex: 1 1
flex: 2 0
flex: 1 1
```

Reply:
Maybe this is because the nw version is 0.51.1 (Linux) vs 0.60.0 (Windows)?

The third parameter is the `flex-basis`. If you don't specify it is like `auto`. Try to add an aditional parameter `flex-basis: 360px;` for example, to see if it accepts it.

Reply:
Indeed flex-basis parameter is not accepted. Also tried with NW 0.62.0 on Ubuntu and it works.